TFT/src/User/Menu/SendGcode.c

```diff
@@ -1,9 +1,14 @@
 #include "SendGcode.h"
 #include "includes.h"
-#if (defined(TFT28_V3_0) || defined(TFT24_V1_1) || defined (MKS_32_V1_4_NOBL))
-  #define TERMINAL_MAX_CHAR ((LCD_WIDTH / BYTE_WIDTH) * (LCD_HEIGHT / BYTE_HEIGHT) * 6)  // for lower RAM 
+/*#if ((defined(MKS_28_V1_0)) || (defined(MKS_32_V1_4) && !defined(MKS_32_V1_4_NOBL)))
```

Comment:
Because the commented part is replaced by the following lines. This is a test PR, I am waiting for feedback from others since I cannot test onboard SD. Once it is proven to resolve the intended issue I will clean it up and make another commit.

Reply:
Okay. Thank you for the explanation..